doc/source/extending-jupyterhub.rst

```diff
@@ -17,12 +17,22 @@ The general method is:
 
      .. code-block:: bash
 
-        helm upgrade <YOUR_RELEASE_NAME> https://github.com/jupyterhub/helm-chart/releases/download/v0.3/jupyterhub-v0.3.tgz -f config.yaml
+        helm upgrade <YOUR_RELEASE_NAME> https://github.com/jupyterhub/helm-chart/releases/download/<HELM_CHART_VERSION>/jupyterhub-<HELM_CHART_VERSION>.tgz -f config.yaml
 
    Where ``<YOUR_RELEASE_NAME>`` is the parameter you passed to ``--name`` when
    `installing jupyterhub <setup-jupyterhub.html#install-jupyterhub>`_ with
-   ``helm install``. If you don't remember it, you can probably find it by doing
+   ``helm install`` and ``<HELM_CHART_VERSION>`` is the desired version.
```

Comment:
This will actually change a bit more for 0.4, I'll make the changes. Plus IMO we should explicitly mention the version here...